lib/bolt/transport/orch.rb

```diff
@@ -23,7 +24,7 @@ def create_client
       end
 
       def build_request(targets, task, arguments)
-        { task: task_name_from_path(task),
+        { task: task_name_from_path(task.executable),
```

Comment:
If we're passing around the Task instance, can't we just take the name from that instead of parsing the filename?

Reply:
Unfortunately Puppet's name for a task is weird (i.e. `echo/task/init`). I'll look at simplifying it a little bit with name though.